src/interpreter/map-validator-advanced.test.ts

```diff
@@ -615,5 +615,118 @@ describe('MapValidatorAdvanced', () => {
 
       valid(profileAst, [mapAst]);
     });
+
+    describe('sendgrid update templete', () => {
```

Comment:
can you do this more.. succint? no reason to have five properties here, minimal reproduction should be enough :sweat_smile: